src/App.tsx

```diff
@@ -15,36 +18,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const IS_SORTED_ALPHABETICALLY = 'alphabetically';
+const IS_SORTED_BY_LENGTH = 'length';
+
+function getPreparedGoods(
+  goods: string[],
+  { sortType, isReversed }: { sortType: string; isReversed: boolean },
+) {
+  let preparedGoods = [...goods];
+
+  switch (sortType) {
+    case IS_SORTED_ALPHABETICALLY:
+      preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+      break;
+
+    case IS_SORTED_BY_LENGTH:
+      preparedGoods.sort((good1, good2) => good1.length - good2.length);
+      break;
```

Comment:
Try to make it shorter, you wrote `preparedGoods.sort((good1, good2) =>` 2 times